bootstrap/bootstrap.go

```diff
@@ -1035,6 +1063,15 @@ func (b *Bootstrap) Start() error {
 		}
 
 		if b.GitSubmodules {
+			// submodules might need their fingerprints verified too
+			if b.SSHFingerprintVerification {
+				commentf("Checking to see if submodule urls need to be added to known_hosts")
+				submoduleRepos, _ := b.gitEnumerateSubmoduleURLs()
```

Comment:
We should probably show a warning if it fails for what ever reason I think, even if we just show it an continue on our merry way.